aptos-move/mvhashmap/src/types.rs

```diff
@@ -213,6 +177,7 @@ impl<V: TransactionWrite> ValueWithLayout<V> {
 #[derive(Clone, Debug)]
 pub enum UnknownOrLayout<'a> {
     Unknown,
+    // TODO: Make this Arc<MoveTypeLayout> to avoid deep cloning.
```

Comment:
Thanks, this is actually quite stupid: we have arcs, but pass them as raw `&` here and do a clone 🤦‍♂️but let's do this change separately.

Reply:
Added to the V-team todos xD